CONTRIBUTING.md

```diff
@@ -61,7 +61,18 @@ There are also some integration tests, which can be run from the Debug pane, or
 
 Tests will output log output to `./.test-reports/testLog.log` for debugging
 
-<!--#### Common Issues-->
+#### Run a specific test
```

Comment:
Two things:
1. If you put this in a separate PR, we can merge it sooner rather than later.
2. If we add this to the docs, we should definitely add a lint rule around this to ensure it doesn't make it into master accidentally.

Reply:
Also note: https://github.com/aws/aws-toolkit-vscode/pull/973 might make this moot, or at least significantly less important to note.